### ISSUE TITLE
Choose today by default

### ISSUE THREAD

**Issue by johntyree** [andrayantelo/calendar-js#47]
There's no reason to make me click `set` if the date is already chosen as today. I can always choose a new date if I want to.

**Comment by andrayantelo** [comment]
What about if you were working on a calendar, and you saved it, then you reload... you'd want your saved calendar to appear right? Not just a calendar that starts with the current day?

**Comment by johntyree** [comment]
Right, I'm arguing here that it should never start with a blank calendar.
